# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: editor/libeditor/HTMLEditor.cpp
# Commit: 0bffa7979736
# Full Hash: 0bffa7979736865a3e660f6d676ac829881d7b01
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2021-07-13 21:42:32
# Regressor Bug: 1718924
# File Overlap Count: 2
# Description:
#   Bug 1718924 - part 13: Move `TextServicesDocument::NodeHasOffsetEntry()` into `OffsetEntryArray` r=m_kato
#   
#   Depends on D119159
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D119160
# ==============================================================================

diff -r 622d9feefc23 -r 0bffa7979736 editor/libeditor/HTMLEditor.cpp
--- a/editor/libeditor/HTMLEditor.cpp	Tue Jul 13 10:19:40 2021 +0000
+++ b/editor/libeditor/HTMLEditor.cpp	Tue Jul 13 10:46:40 2021 +0000
@@ -4637,7 +4637,8 @@
 
   if (mTextServicesDocument && NS_SUCCEEDED(rv)) {
     RefPtr<TextServicesDocument> textServicesDocument = mTextServicesDocument;
-    textServicesDocument->DidJoinNodes(aLeftNode, aRightNode);
+    textServicesDocument->DidJoinNodes(*aLeftNode.AsContent(),
+                                       *aRightNode.AsContent());
   }
 
   if (!mActionListeners.IsEmpty()) {